File: 백준/Bronze/10818. 최소， 최대/최소， 최대.cc
#include <iostream>
using namespace std;

int main()
{
    long int N;
    cin>>N;
    long int arr[N];
    for(int i=0; i<N; i++)
    {
        cin>>arr[i];
    }
    long int max = arr[0];
    long int min = arr[0];
    for(int j=0; j<N; j++)
    {
        if(max<arr[j]) max=arr[j];
    }
    for(int k=0; k<N; k++)
    {
        if(min>arr[k]) min=arr[k];
    }
    cout<<min<<" "<<max<<endl;

    return 0;
}

File: 백준/Bronze/10818. 최소， 최대/최소， 최대.java
import java.io.BufferedReader;
import java.io.BufferedWriter;
import java.io.IOException;
import java.io.InputStreamReader;
import java.io.OutputStreamWriter;
import java.util.Collections;
import java.util.StringTokenizer;
import java.util.Vector;

public class Main {

  public static void main(String[] args) throws IOException {
    BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
    BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));
    int N = Integer.parseInt(br.readLine());
    StringTokenizer st = new StringTokenizer(br.readLine());
    br.close();
    Vector<Integer> v = new Vector<>();
    for (int i = 0; i < N; i++) { v.add(Integer.parseInt(st.nextToken())); }
    int max = Collections.max(v);
    int min = Collections.min(v);
    bw.write(String.valueOf(min) + " " + String.valueOf(max));
    bw.flush();
    bw.close();
  }
}